### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by Jermolene** [38911BytesFree/TiddlyWiki#2]
Hi @38911BytesFree I wanted to say thank you. I only just found your work, and was thrilled to see that you've worked out a solid mechanism for saving directly to GitHub (I had not realised that it was possible). I'm now adapting your code for TW5, and I think it will be much appreciated by a lot of people.

**Comment by communicotim** [comment]
I agree, there are some amazing possibilities. Also for projects like [olelo](https://github.com/minad/olelo) or [gollum](https://github.com/gollum/gollum). I'm amazed that I'd never heard of doing this before.

**Comment by 38911BytesFree** [comment]
Thanks ! Glad to hear it was useful for you :)
